docs/custom_template/partials/head.tmpl.partial

```diff
@@ -16,7 +16,17 @@
   <meta name="title" content="{{#title}}{{title}}{{/title}}{{^title}}{{>partials/title}}{{/title}} {{#_appTitle}}| {{_appTitle}} {{/_appTitle}}">
   <meta name="generator" content="docfx {{_docfxVersion}}">
   {{#_description}}<meta name="description" content="{{_description}}">{{/_description}}
-  <link rel="shortcut icon" href="{{_rel}}{{{_appFaviconPath}}}{{^_appFaviconPath}}favicon.ico{{/_appFaviconPath}}">
+
+  <link rel="apple-touch-icon" sizes="180x180" href="{{_rel}}{{{_appFaviconPath}}}{{^_appFaviconPath}}favicons/apple-touch-icon.png{{/_appFaviconPath}}">
```

Comment:
👍 Nice technique here! Taught me something.

Reply:
You are welcome